modules/setup.rb

```diff
@@ -1,15 +1,17 @@
-module YuukiBot
+# frozen_string_literal: true
 
+module YuukiBot
   class << self
-     attr_accessor :config
+    attr_accessor :config
   end
 
   require 'yaml'
   # Load Config from YAML
 
-  unless File.exists?('config/config.yml')
+  unless File.exist?('config/config.yml')
     puts 'You don\'t have a valid config file!'
-    puts 'If you want to create a config file manually, please stop this program with Ctrl+C and follow the instructions in config/README.md.'
+    puts 'If you want to create a config file manually, please stop this program with Ctrl+C ' \
+      'and follow the instructions in config/README.md.'
     puts 'Waiting 3 seconds...'
     sleep 3
```

Comment:
Can we just make the user press enter here? Seems a bit rude to go YOU HAVE 3 SECONDS TO CANCEL BEFORE WE THROW INSTRUCTIONS AT YOU.